sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/ApproximatePercentile.scala

```diff
@@ -51,7 +51,7 @@ import org.apache.spark.sql.types._
   usage = """
     _FUNC_(col, percentage [, accuracy]) - Returns the approximate percentile value of numeric
       column `col` at the given percentage. The value of percentage must be between 0.0
-      and 1.0. The `accuracy` parameter (default: 10000) is a positive numeric literal which
+      and 1.0. The `accuracy` parameter (default: 100000) is a positive numeric literal which
```

Comment:
I am not sure about this. The default is already documented and it's already able to configure. I wouldn't change this unless there's a strong reason to change its default.

Reply:
how to configure

Reply:
You can set `accuracy` parameter

Reply:
I found that I can change the parameters when I use, but summary() will call the default value. Can we configure the default value in the form of spark Config?

Reply:
It's just an approximate to summarize the current Dataset. If you need more accurate number, you can just manually call this function.

Reply:
OK , thanks